### PR TITLE
move to use cluster-logging-load-client

### PR DESCRIPTION
**Pull request by eranra**

### CODE REVIEW

File: config/development.yaml [hunk 1]
```diff
@@ -25,65 +25,8 @@ loki:
...
-            fullstackend: 'sum(rate({component="fullstackend"}[1m])) by (level)'
-            frontend: 'sum(rate({component="frontend"}[1m])) by (level)'
-            backend: 'sum(rate({component="backend"}[1m])) by (level)'
-            allpanics: 'sum(rate({msg=~"panic.*"}[1m]))'
-            countpanics: 'count_over_time({msg=~"panic.*"}[5m])'
-            topTenErrors: 'topk(10, sum(rate({component!="", level="error"}[1h])) by (component))'
-          startThreshold: 1024000
   highVolumeWrites:
-    enabled: true
+    enabled: false
```
Comment (Red-GV) [inline config/development.yaml:29]: Should be true?

File: internal/config/config.go [hunk 2]
```diff
@@ -93,13 +93,15 @@ func (lc *Loki) QueryRangeURL() string {
...
 }
 
 type Readers struct {
-	Replicas         int32             `yaml:"replicas"`
-	Queries          map[string]string `yaml:"queries"`
-	StartThreshold   float64           `yaml:"startThreshold"`
-	QueriesPerSecond int32             `yaml:"throughput"`
+	Replicas       int32             `yaml:"replicas"`
+	Args           map[string]string `yaml:"args"`
+	Command        string            `yaml:"command"`
```
Comment (Red-GV) [inline internal/config/config.go:102]: Is there a list of this commands somewhere?

File: config/development.yaml (hunk 1 above)
Comment (eranra) [reply config/development.yaml:29]: Yes ... let me fix

Comment (eranra) [reply config/development.yaml]: Fixed :-)

File: internal/config/config.go (hunk 2 above)
Comment (eranra) [reply internal/config/config.go:102]: two options "generate" and "query"

Comment (Red-GV) [reply internal/config/config.go:102]: Maybe add to readme or as a comment above this?

**Review by periklis — COMMENTED** [review]
Looks good to me

**Pull request merged**